charts/newrelic-infrastructure/templates/ksm/_agent-config_helper.tpl

```diff
@@ -0,0 +1,20 @@
+{{- /*
+Defaults for ksm's agent config
+*/ -}}
+{{- define "nriKubernetes.ksm.agentConfig.defaults" -}}
+is_forward_only: true
+is_secure_forward_only: true
+overide_host_root: ""  # Typo from here: https://github.com/newrelic/infrastructure-agent/blob/master/pkg/config/config.go#L267
```

Comment:
Can you explain further?

Reply:
As I said in other thread:

> Is also a step to remove the forwarder image and use agent one directly if we want to remove spurious hard-to-follow dependencies.

I used [the Dockerfile from the forwarder as an example](https://github.com/newrelic/infrastructure-agent/blob/master/build/container/Dockerfile#L47-L66) to set these environment variables (even if I am still using the forwarder image).

I am closing this thread and we can do the follow-up [here](https://github.com/newrelic/nri-kubernetes/pull/383#discussion_r845220611) if you would.